homework5.1/task1.js

```diff
@@ -0,0 +1,17 @@
+const averageGrade = 90;
+
+if(averageGrade < 60){
+    console.log('Незадовільно')
+}
+else if(averageGrade >= 60 && averageGrade <= 70){
```

Comment:
averageGrade >= 60 зайвий, вже знаємо з минулого іфа. Тобто, якщо наша програма і заходить в цей іф, то ця умова буде завжди виконуватись
```suggestion
else if(averageGrade <= 70){
```